apps/dialtone-documentation/docs/design/typography/index.md

```diff
@@ -1,89 +1,729 @@
 ---
 title: Typography
-status: planned
-description: Guidance for clear, legible, and easy-to-read text.
+description: This plays a crucial role in establishing visual hierarchy, readability, and brand identity in our digital products.
 ---
 
-Dialtone provides a set of purposeful typographic styles to present user interface and content as clearly and efficiently as possible.
-
-<!-- <div style="background-color: orangered; font-size: 54px">
-  <a href="https://www.figma.com/file/nnMW63EwQxLMJBFLd1qgjo/DAH-September-2022?node-id=3%3A1238&viewport=-4020%2C688%2C0.5&t=VeMOh5CN6E6agKK7-11">https://www.figma.com/file/nnMW63EwQxLMJBFLd1qgjo/DAH-September-2022?node-id=3%3A1238&viewport=-4020%2C688%2C0.5&t=VeMOh5CN6E6agKK7-11</a>
-</div> -->
-
-## Text styles
-
-<aside class="d-notice d-notice--info d-mt24 d-wmx100p" role="status" aria-hidden="false">
-  <div class="d-notice__icon">
-    <dt-icon name="info"></dt-icon>
-  </div>
-  <div class="d-notice__content d-stack4">
-    <p class="d-notice__message">
-      <strong>WIP</strong>
-    </p>
-  </div>
-</aside>
-
-## Font family
-
-<aside class="d-notice d-notice--info d-mt24 d-wmx100p" role="status" aria-hidden="false">
-  <div class="d-notice__icon">
-    <dt-icon name="info"></dt-icon>
-  </div>
-  <div class="d-notice__content d-stack4">
-    <p class="d-notice__message">
-      <strong>WIP</strong>
-    </p>
-  </div>
-</aside>
-
-## Font size
-
-<aside class="d-notice d-notice--info d-mt24 d-wmx100p" role="status" aria-hidden="false">
-  <div class="d-notice__icon">
-    <dt-icon name="info"></dt-icon>
-  </div>
-  <div class="d-notice__content d-stack4">
-    <p class="d-notice__message">
-      <strong>WIP</strong>
-    </p>
-  </div>
-</aside>
-
-## Font weight
-
-<aside class="d-notice d-notice--info d-mt24 d-wmx100p" role="status" aria-hidden="false">
-  <div class="d-notice__icon">
-    <dt-icon name="info"></dt-icon>
-  </div>
-  <div class="d-notice__content d-stack4">
-    <p class="d-notice__message">
-      <strong>WIP</strong>
-    </p>
-  </div>
-</aside>
-
-## Line height
-
-<aside class="d-notice d-notice--info d-mt24 d-wmx100p" role="status" aria-hidden="false">
-  <div class="d-notice__icon">
-    <dt-icon name="info"></dt-icon>
-  </div>
-  <div class="d-notice__content d-stack4">
-    <p class="d-notice__message">
-      <strong>WIP</strong>
-    </p>
-  </div>
-</aside>
-
-## Typescale
-
-<aside class="d-notice d-notice--info d-mt24 d-wmx100p" role="status" aria-hidden="false">
-  <div class="d-notice__icon">
-    <dt-icon name="info"></dt-icon>
-  </div>
-  <div class="d-notice__content d-stack4">
-    <p class="d-notice__message">
-      <strong>WIP</strong>
-    </p>
-  </div>
-</aside>
+<svg width="746" height="301" viewBox="0 0 746 301" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Can we isolate this svg, and other large svgs in it's own file? It's reallllly big. See `SvgLoader.vue`